test/fixtures/e2e/bento/lightbox.html

```diff
@@ -6,11 +6,6 @@
       async
       src="https://cdn.ampproject.org/v0/bento-lightbox-1.0.js"
     ></script>
-    <link
```

Comment:
Same here, why is this ok to remove?

Reply:
hmm didn't mean to remove this, i can revert